### ISSUE TITLE
Can't access to the admin page in voyager 1.0

### ISSUE THREAD

**Issue by ngomtanouba** [thedevdojo/voyager#1886]
- Laravel Version: 5.5.13
- Voyager Version: 1.0
- PHP Version:7.0.10
- Database Driver & Version:MySql 5.7.14

### Description:
Hello!
After `composer  require tcg/voyager`  i run `php artisan voyager:install --with-dummy` .

All things are correct but at the end of my installation i got this issue:

`Attempting to set Voyager User model as parent to App\User
Dumping the autoloaded files and reloading all new files

  [Symfony\Component\Process\Exception\ProcessTimedOutException]
  The process "composer dump-autoload" exceeded the timeout of 60 seconds.`

And then can't to my admin page.
Need your help! Thanks

**Comment by ngomtanouba** [comment]
@fletch3555 I delete wich one the public vendor directory or the '/vendor/tcg ? I've ran `composer dump-autoload` then  `php artisan voyager:install` but return `[Symfony\Component\Process\Exception\ProcessTimedOutException]
  The process "composer dump-autoload" exceeded the timeout of 60 ` again

**Comment by Maghraby** [comment]
@fletch3555 the error because can't load jquery.
![Uploading Screen Shot 2017-09-27 at 8.12.34 PM.png…]()

**Comment by Maghraby** [comment]
@fletch3555 this's because can't load jquery
<img width="1436" alt="screen shot 2017-09-27 at 8 12 34 pm" src="https://user-images.githubusercontent.com/7889084/30930556-5242d668-a3c2-11e7-97ef-ce1dc170186e.png">

**Comment by ngomtanouba** [comment]
What can we do in this case? @Maghraby

**Comment by fletch3555** [comment]
@Maghraby, the presence or absence of jquery has absolutely nothing to do with this issue.

@BarthFox, I meant delete `/vendor/tcg/voyager`, not `/public/vendor`.

**Comment by Maghraby** [comment]
@fletch3555 jquery can't load any data
<img width="1440" alt="screen shot 2017-09-30 at 6 38 40 pm" src="https://user-images.githubusercontent.com/7889084/31047649-b07271aa-a60e-11e7-9635-33481266c26f.png">

**Comment by ngomtanouba** [comment]
Hi @fletch3555  Even i delete the vendor folder it returns the same error. i installed the new version of laravel with the new version of voyager but nothing changes

**Comment by yakuter** [comment]
I have the same problem and still couldn't find the solution.

**Comment by ngomtanouba** [comment]
Even now i can't resolve it

**Comment by yakuter** [comment]
I solved the problem. Its all about the "**browser cache**". Just clear history, that's all.

**Comment by mpenna** [comment]
Tried on two separate vagrant homestead environments: 
- Ubuntu 16.04 / PHP 7.1.0 / L 5.5
- Ubuntu 14.04 / PHP 7.0.5 / L 5.4

On both environments, the "artisan voyager:install" command hangs during the "Dumping the autoloaded files and reloading all new files" phase. 

Result: no /admin route is available. These are the routes that show when issuing an "artisan route:list" command: 

GET|HEAD | /                              | Closure                                                 | web
GET|HEAD | api/user                  | Closure                                                 | api,auth:api
GET|HEAD | arrilot/load-widget | Arrilot\Widgets\Controllers\WidgetController@showWidget |

@yakuter I can't possibly imagine why this would have anything to do with the browser's history cache.

**Comment by fletch3555** [comment]
@mpenna, please make sure VoyagerServiceProvider is loaded (either statically (5.4) or dynamically (5.5)).  Sounds like `artisan voyager:install` isn't writing the routes to the routes file.  Do you have permissions set up correctly?

**Comment by mpenna** [comment]
Hi @fletch3555  Voyager's service provider is seems to be loading OK. Debugging the install command, I was able to detect that the installation hangs during the dump-autoload phase. 

$process = new Process($composer.' dump-autoload');

Just as a test, I commented out that line and once that was out of the equation the installation ran all the way through without issues doing what it is supposed to do, which includes writing Voyager's routes to app's web routes file.

Some other wierdness is going on here, that's for sure.

**Comment by mpenna** [comment]
The actual line that was commented out is this one:

$process->setWorkingDirectory(base_path())->run();

**Comment by mpenna** [comment]
I guess I've found out what the problem is. 

I have brought that line back into the picture but first I cancelled the timeout setting for the corresponding process:

```
$process = new Process($composer.' dump-autoload');
$process->setTimeout(null);
$process->setWorkingDirectory(base_path())->run();
```

With this in place, the voyager:install command ran without issues. Apparently, here in my environment the "composer dump-autoload" needs a bit more of time to fully run.

**Comment by nmduarte** [comment]
@mpenna thanks for this! removing the timeout worked for me. guess my "composer dump-autoload" needs a bit more time to run as well.

**Comment by ekfinbarr** [comment]
`$process = new Process($composer.' dump-autoload');
$process->setTimeout(null);
$process->setWorkingDirectory(base_path())->run();`

@mpenna Which file will I make these changes?

**Comment by Aliji-Emmanuel** [comment]
`$process = new Process($composer.' dump-autoload'); $process->setTimeout(null); $process->setWorkingDirectory(base_path())->run();`
these lines are found in vendor/symphony/process/Process.php at line 165
it is written as  `$this->setTimeout(null);`, note that i have already changed my value to null, you should yours should be 60 if it is unchanged.

**Comment by emptynick** [comment]
This was fixed in #2552
